Apps/Src/Task_VoltTempMonitor.c

```diff
@@ -90,6 +90,13 @@ void Task_VoltTempMonitor(void *p_arg) {
                         OS_OPT_POST_1,
                         &err);
             assertOSError(err);
+            //suggest that the battery should not be charged
+            CanMsg.id = CHARGE_ENABLE;
+            CanPayload.idx = 0;
+            CanData.b = 0;
+            CanPayload.data = CanData;
+            CanMsg.payload = CanPayload;
+            OSQPost(&CANBus_MsgQ, &CanMsg, sizeof(CanMsg), OS_OPT_POST_FIFO, &err);
```

Comment:
This code will never run because posting the fault semaphore will call the Fault State task and kill all of the systems that are using this message (controls and array). We should instead be sending this signal when the highest temperature recorded in the battery box is between 45C and 60C (possibly with some tolerance on the lower end)